package network.cere.ddc.client.common.exception

class InitializeException(cause: Throwable) : RuntimeException(cause)